### PR TITLE
Create Grid Unique Paths

### PR DESCRIPTION
**Pull request by kavya-gupta18**

### CODE REVIEW

**Comment by kavya-gupta18** [comment]
Hey, I have added this grid's unique paths. it would be great for me if merge it and add the hacktoberfest2022 label to it.

**Review by sam-tripathi — CHANGES_REQUESTED** [review]
Could you add some explanatory part in  form  of comments  and the output too for better understanding of beginners

**Comment by kavya-gupta18** [comment]
now check for the changes

File: DataStructures/Grid Unique Paths [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    int uniquePaths(int m, int n) {
+        vector<vector<int>> dp(m+1,vector<int>(n+1,-1));
+       
+        //dp[1][1]=1;
+       int num=countPaths(0,0,m,n,dp);
+        if(m==1&&n==1)
+            return num;
+        return dp[0][0];
+    }
+};
```
Comment (sam-tripathi) [inline DataStructures/Grid Unique Paths:44]: error

**Comment by sam-tripathi** [comment]
Ok

**Pull request merged**